catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -75,6 +81,8 @@ const SearchForm = ({
   shouldShowFilters,
   searchParams,
   workTypeAggregations,
+  placeholder,
+  itemsUrl = worksUrl,
```

Comment:
This confuses me a bit - could we have something like `formUrl`or just `url`?

Reply:
I say this because `items` is overloaded - I am assuming you meant that `items` is a superset of `works` and `images`?

Reply:
yep, naming issue, don't think I knew about `items` at this point